app/layout.tsx

```diff
@@ -19,6 +19,32 @@ export const metadata: Metadata = {
   description:
     "Claim your vote tokens, cast your vote and track progress on SIMD-0326: Alpenglow",
   metadataBase: new URL("https://gov.blueshift.gg"),
+  openGraph: {
+    title: "SIMD-0326: Alpenglow - Blueshift Governance Dashboard",
+    description:
+      "Claim your vote tokens, cast your vote and track progress on SIMD-0326: Alpenglow",
```

Comment:
Not needed. `title` and `description` in the outer object is automatically injected for OpenGraph